src/codegen/codegen_acc_visitor.cpp

```diff
@@ -5,8 +5,9 @@
  * Lesser General Public License. See top-level LICENSE file for details.
  *************************************************************************/
 
+#include "fmt/format.h"
```

Comment:
since these are external dependencies, can we not include via `<>` and "-System" to avoid the compiler stumbling over warnings that don't matter to us?

Reply:
yeah agree. I used the "existing" style -  which we should change that consistently.
As there are multiple external submodules included like this, I created separate issue #269 where we can change that everywhere.